pyibisami/ami_model.py

```diff
@@ -304,13 +304,13 @@ def initialize(self, init_object):
                 self._num_aggressors,
                 self._sample_interval,
                 self._bit_time,
-                "{}".format(self._ami_params_in),  # Prevents model from mucking up our input parameter string.
+                self._ami_params_in,
                 byref(self._ami_params_out),
                 byref(self._ami_mem_handle),
                 byref(self._msg),
             )
-        except:
-            raise
+        except OSError as error:
```

Comment:
It helps me when testing or porting to know what it should raise.  Instead of guessing on the different types of data that you will get or have planned on.  I'm also using pylint to lint your code as I correct some of it.  It will complain about this.   `tox -e pylint`